recipes/libmysqlclient/all/conanfile.py

```diff
@@ -143,7 +145,7 @@ def _configure_cmake(self):
 
         if self.options.with_zlib:
             self._cmake.definitions["WITH_ZLIB"] = "system"
-
+ 
```

Comment:
```suggestion

```
You can remove this now.

Reply:
Thanks, coding on mobile is ... tiring

Reply:
Haha, indeed.
My suggestion was made on mobile as well 🤣

Reply:
Perhaps, instead of introducing dumb changes like this,
we add a comment at the top of the recipe with a counter.
e.g.:
```python
from conans import ConanFile, tools
import os

# c3i builds regeneration counter: 1 (bump this number when binaries are outdated)

...
```

Reply:
Actually my intent was to fix the apple m1 case from the beginning, not regenerate outdated binaries